src/main/java/at/laborg/briss/BrissSwingGUI.java

```diff
@@ -563,19 +603,18 @@ public void setPositionSelRects() {
         if ((minX < Integer.MAX_VALUE) && (minY < Integer.MAX_VALUE)) {
             minX = Math.round(25.4f * minX / 72f);
             minY = Math.round(25.4f * minY / 72f);
-            defInput = minX + " " + minY; //$NON-NLS-1$
+            defInput = minX + " " + minY; // $NON-NLS-1$
         }
 
         // get user input
         // minimums are used as a default
         String input = JOptionPane.showInputDialog(Messages.getString("BrissGUI.position"), defInput);
 
-        if (input == null || input.equals("")) //$NON-NLS-1$
-            return;
+        if (input == null || input.equals("")) // $NON-NLS-1$
+        return;
```

Comment:
Same as before, maybe we should add the parenthesis so the formatter can also understand it better